packages/common/src/hooks/useChangeEmailFormConfiguration.ts

```diff
@@ -61,9 +60,9 @@ const confirmPasswordFormikSchema = toFormikValidationSchema(
 const verifyEmailFormikSchema = toFormikValidationSchema(confirmEmailSchema)
 
 export const useChangeEmailFormConfiguration = (onComplete: () => void) => {
-  const { audiusBackend } = useAppContext()
   const [page, setPage] = useState(ChangeEmailPage.ConfirmPassword)
   const audiusQueryContext = useAudiusQueryContext()
+  const { authService } = audiusQueryContext
```

Comment:
nit: why not `const {authService} = useAudiusQueryContext()`?

Reply:
cause `audiusQueryContext` is also passed into `emailSchema`